core/google/cloud/_http.py

```diff
@@ -279,7 +279,9 @@ def api_request(self, method, path, query_params=None,
             can allow custom behavior, for example, to defer an HTTP request
             and complete initialization of the object at a later time.
 
-        :raises: Exception if the response code is not 200 OK.
+        :raises ~google.cloud.exceptions.GoogleCloudError: if the response code
+            is not 200 OK.
+        :raises TypeError: if the response content type is not JSON.
```

Comment:
@jonparrott I've tried to fight `pylint` on this error before and gave up.

I did because

1. There were too many
1. It seemed I couldn't get them to render the way I hoped